src/java/htsjdk/samtools/CRAMFileReader.java

```diff
@@ -244,26 +277,26 @@ public SAMRecordIterator assertSorted(final SortOrder sortOrder) {
         if (filePointers == null || filePointers.length == 0)
             return emptyIterator;
 
-        final SeekableStream s = getSeekableStreamOrFailWithRTE();
-        final CRAMIterator si;
-        try {
-            s.seek(0);
-            si = new CRAMIterator(s, referenceSource);
-            si.setValidationStringency(validationStringency);
-            it = si;
-        } catch (final IOException e) {
-            throw new RuntimeEOFException(e);
-        }
-
         Container c;
```

Comment:
Variable names like ```c``` and ```s``` make it extremely hard to follow the code, as we have to keep going back its declaration to know what they are.  Could you please go back through the code and change variable names that reference objects to be more descriptive (ex. ```stream``` or ```container```).